ros2topic/ros2topic/verb/info.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ros2cli.node.direct import DirectNode
+from ros2topic.api import TopicNameCompleter
+from ros2topic.verb import VerbExtension
+
+
+class InfoVerb(VerbExtension):
+    """Print information about a topic."""
+
+    def add_arguments(self, parser, cli_name):
+        arg = parser.add_argument(
+            'topic_name',
+            help="Name of the ROS topic to get info (e.g. '/chatter')")
+        arg.completer = TopicNameCompleter(
+            include_hidden_topics_key='include_hidden_topics')
+
+    def main(self, *, args):
+        with DirectNode(args) as node:
+            topic_name = args.topic_name
+            count_publishers = node.count_publishers(topic_name)
+            count_subscribers = node.count_subscribers(topic_name)
+            print('Topic: %s' % topic_name)
+            print('Publishers count: %d' % count_publishers)
+            print('Subscribers count: %d' % count_subscribers)
```

Comment:
Should be "Publisher count" / "Subscriber count" (singular instead of plural).

Reply:
Fixed.